src/Migration/Version3100.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Terramar\Packages\Migration;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Migration script for 3.0->3.1
+ *
+ * Only necessary if upgrading from 3.0 to 3.1.
+ *
+ * @version 3.1.0
+ */
+class Version3100 extends AbstractMigration
```

Comment:
This convention is a holdover from when Packages was an internal Terramar Labs project. We deployed based on versions, and we always have one migration per version, so we opted to name the migrations after the version they represented. This convention is still compatible with doctrine-migrations, represented via: XY0Z where X is major, Y is minor, and 0Z is the patch level with 0 prepended if patch < 10.